smac/scenario/scenario.py

```diff
@@ -345,6 +358,57 @@ def extract_instance_specific(instance_list):
         else:
             self.logger.info("Output to %s" % (self.output_dir))
 
+    def _write(self):
+        """Write scenario to file in a format that can be easily retrieved using
+        the input_reader. Will overwrite if file already exists.
+
+        Sideeffect: creates output-directory if it doesn't exist.
+        """
+        self.logger.debug("Writing scenario-file to {}.".format(self.output_dir))
+        # Reverse mapping of option-names to internal -> external
+        options_int2ext = {v: k for k, v in self.options_ext2int.items()}
```

Comment:
Does a scenario file written for logging purposes has to be backwards compatible?